README.md

```
@@ -346,62 +378,66 @@ This repository only contains the source code for the package.
-        ./gradlew clean test
-    ```
+   ```shell script
+       ./gradlew clean test
+   ```
 
 3. To run a group of tests
-    ```
-    ./gradlew clean test -Pgroups=<test_group_names>
-    ```
+
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)